### PR TITLE
Add footer to README, with License, CoC and Contrib

### LINKED ISSUE #5: Choose a license for the repository
Quoting @smola https://github.com/src-d/metadata-retrieval/pull/4#issuecomment-534955246
> I think we'll need Apache 2.0 license here (to import on proprietary code)

### PR DESCRIPTION
**Pull request by dpordomingo**
fix #5

- Add common footer into `README.md` with License, company CoC and company Contrib
- Add `LICENSE.md` with `Apache 2.0` as required by #5, caused by https://github.com/src-d/metadata-retrieval/pull/4#issuecomment-534955246

### CODE REVIEW

**Comment by dpordomingo** [comment]
Waiting for @smola confirmation, who should approve the license.

**Review by se7entyse7en — APPROVED** [review]

**Comment by dpordomingo** [comment]
Rebased over current `master`.
@se7entyse7en feel free to merge once @smola approves

**Comment by se7entyse7en** [comment]
Blocked by [this](https://github.com/src-d/minutes/pull/766/files#diff-188cb21a5be8beb2ee0adb7df1a3e635R38).

**Comment by se7entyse7en** [comment]
See [here](https://github.com/src-d/metadata-retrieval/issues/5#issuecomment-545116726). @dpordomingo once rebased feel free to merge it.

**Comment by dpordomingo** [comment]
I also added `NOTICE.md`, `DCO` and `.github/CODE_OF_CONDUCT.md` from our engineering conventions.

**Pull request merged**
